### ISSUE TITLE
No firefox support

### ISSUE THREAD

**Issue by sdsoldi** [takuma-ru/vue-swipe-modal#94]
"@takuma-ru/vue-swipe-modal": "^5.0.1"
"nuxt": "^3.10.3"

Console error 'Uncaught (in promise) DOMException: Element.animate: '::backdrop' is an unsupported pseudo-element.'

**Comment by takuma-ru** [comment]
@sdsoldi 
Upon investigation, we found the following
There appears to be an error in the handling of `backdrop` animations.

There is a way to replace the `backdrop` with a div, but we do not want to stop using the powerful `::backdrop` to fix a less important animation bug.
So, as a temporary measure, I have decided to disable the animation in the affected area.
We will fix this at a later date.

https://github.com/takuma-ru/vue-swipe-modal/blob/26da1743cd80f5bceff3a0c0e001af368aab7462/packages/core/src/components/SwipeModal.vue#L408-L412
